### DISCUSSION TITLE
How much GPU memory is required to perform MD simulations with millions of atoms?

### DISCUSSION THREAD

**Discussion by DingChangjie** [deepmodeling/deepmd-kit#3081] [Q&A] [answered]
I'm trying to perform radiation damage simulations (i.e., cascade simulations) with a compressed deep potential. Typically such a system contains millions of atoms. How much GPU memory is required to perform such a simulation? Could anyone recommend suitable GPU? Thanks

**Comment by Taixi-Feng** [comment]
Try to use a DP-compress model. This reduces computational costs, but there may be issues with model unavailability. I ran a about 10,000 atom simulation system using less than 2000MB of memory on the V100s

**Comment by njzjz** [comment, marked as answer]
[Our paper](https://doi.org/10.1063/5.0155600) has some results. I want to note that, with MPI, you do not need to run simulations on only one GPU.
